lib/autoproj/autobuild_extensions/dsl.rb

```diff
@@ -164,8 +164,11 @@ def import_package(name, workspace: Autoproj.workspace, &block)
     package_common(:import, name, workspace: Autoproj.workspace, &block)
 end
 
-def python_package(name, workspace: Autoproj.workspace, &block)
-    package_common(:python, name, workspace: Autoproj.workspace, &block)
+def python_package(name, workspace: Autoproj.workspace)
+    package_common(:python, name, workspace: Autoproj.workspace) do |pkg|
+        pkg.internal_dependencies << 'python'
```

Comment:
I'm not sold on the `internal_dependencies` name. These are basically dependencies required during import, right ? So what about `import_dependencies` or `early_dependencies` ? Something more descriptive of how they're different from the "normal" dependencies.

Reply:
> These are basically dependencies required during import, right?

Not really.. In this particular case, it required for settings up the environment for python source pacakges (so, after import). The general use case I guess is for dependencies of the `Autobuild::Packages` subclasses (autobuild's own dependencies, in a sense. which is why I suggested internal)

Reply:
Also, I don't think it is a good idea to advertise this too much as there are caveats (we assume that these dependencies are always osdeps and that they do not depend on strict package managers). So, generally, the regular user should not use this

Reply:
Fair enough ... Good for `internal` then. Mark them explicitly as `@api private` then.